tests/test_utils.py

```diff
@@ -308,6 +311,24 @@ def _second_callback(result):
     assert second_results == [3]
 
 
+@pytest.mark.asyncio
+async def test_concurrent_runner_cancel():
```

Comment:
A similar test was added [here](https://github.com/elastic/connectors/pull/1477/files#diff-33c13e0b177bacd2f02e29bcb8aea5b49e7ce34901fd8f41fefb65defba1bd33R261): but it's not merged to `main` yet.

Reply:
Thanks, I'll remove this test!